### DISCUSSION TITLE
hydra cli: hydra token user with pkce

### DISCUSSION THREAD

**Discussion by pommelinho** [ory/hydra#2262] [Q&A] [answered]
Is it possible to use the CLI command  hydra token user (see [here](https://www.ory.sh/hydra/docs/cli/hydra-token-user#hydra-token-user)) with PKCE?

Currently it is complaining when i am not passing the --client-secret param. Or am i missing something?

**Comment by aeneasr** [comment, marked as answer]
No, that is currently not supported. It only supports auth code flows with client secrets. Since this is just a command to help get started with a demo, it will probably stay as such!

**Reply by pommelinho** [reply]
ok thanks
